lib/util.ts

```diff
@@ -61,9 +62,9 @@ export const tokenizeClass = (str: string, regexpStr: string): [SetTokens, numbe
       (rs[7] && {
         type: types.RANGE,
         from: (rs[8] || rs[9]).charCodeAt(0),
```

Comment:
does this need the last char from the captured group like `to` does?

Reply:
No, it is there in `to` because `rs[10]` will be `\\]` rather than `]` (similarly for other escaped chars), and so we need the code of the last char. By the looks `(rs[8] || rs[9])` has a similar effect in `from` because `rs[8]` does not include the `\\` if a character is escaped.